code/modules/cargo/bounties/engineering.dm

```diff
@@ -0,0 +1,57 @@
+/datum/bounty/item/engineering/pluoxium_tank
+	name = "Full Tank of Pluoxium"
+	description = "CentCom RnD is researching extra compact internals. Ship us a tank of Pluoxium and you'll be compensated."
+	reward = 7500 // Easy to produce as a supermatter engine byproduct
+	wanted_types = list(/obj/item/tank)
+	var/moles_required = 20 // A full tank is 28 moles, but CentCom ignores that fact.
+
+/datum/bounty/item/engineering/pluoxium_tank/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/item/tank/T = O
+	if(!T.air_contents.gases[/datum/gas/pluoxium])
+		return FALSE
+	return T.air_contents.gases[/datum/gas/pluoxium][MOLES] >= moles_required
+
+/datum/bounty/item/engineering/nitryl_tank
+	name = "Full Tank of Nitryl"
+	description = "The non-human staff of Station 88 has been volunteered to test performance enhancing drugs. Ship them a tank of Nitryl so they can get started."
+	reward = 15000 // Requires actual effort to produce
+	wanted_types = list(/obj/item/tank)
+	var/moles_required = 20
+
+/datum/bounty/item/engineering/nitryl_tank/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/item/tank/T = O
+	if(!T.air_contents.gases[/datum/gas/nitryl])
+		return FALSE
+	return T.air_contents.gases[/datum/gas/nitryl][MOLES] >= moles_required
+
+/datum/bounty/item/engineering/tritium_tank
+	name = "Full Tank of Tritium"
+	description = "Station 49 is looking to kickstart their research program. Ship them a tank full of Tritium."
+	reward = 20000 // Coveted by RnD, very useful for tritium bombs and thus hard to get hold of
+	wanted_types = list(/obj/item/tank)
+	var/moles_required = 20
+
+/datum/bounty/item/engineering/tritium_tank/applies_to(obj/O)
+	if(!..())
+		return FALSE
+	var/obj/item/tank/T = O
+	if(!T.air_contents.gases[/datum/gas/tritium])
+		return FALSE
+	return T.air_contents.gases[/datum/gas/tritium][MOLES] >= moles_required
+
+/datum/bounty/item/engineering/singularity
+	name = "Contained Singularity"
+	description = "Our frontier science division is looking to research wormhole travel. Send us a contained singularity and we'll take it from there."
+	reward = 75000 //requires 14k credits of purchases, not to mention cooperation with engineering/heads of staff to set up inside the cramped shuttle
+	wanted_types = list(/obj/singularity)
+	exclude_types = list(/obj/singularity/energy_ball)
```

Comment:
Considering that singulos eat through the floor will they be sent with the shuttle?


Also this potentially allows selling nar'sie to centcom